CONTRIBUTING.md

```diff
@@ -1,7 +1,71 @@
+# Rationale
+
+Contributions are always welcome. We accept [bug fixes](#bugs),
+[proposals and new ideas](#proposal), or simply [editorial
+improvements](#editorial-improvements).
+
+By submitting a pull request you are implicitly adhering to the
+[contribution agreement](#contribution-agreement)
+
+## Bugs
+
+You can report a bug by creating an
+[issue](https://github.com/ARM-software/acle/issues). You can also fix
+it yourself with a [pull
+request](https://github.com/ARM-software/acle/pulls).
+
+## Proposals
```

Comment:
I wonder if the title could be made a bit more specific/self-describing than "proposals"?
When I see "proposals", I wonder, proposals for what?

Would "Proposals for new content" work?